src/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  background: url("images/star.svg") no-repeat center;
+  margin-right: 4px;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+}
```

Comment:
Strange solution
This `--n` part - is a modifier that define how many stars should be active in this stars list

Instead of applying one rule for 5 classes better to create one class for 5 elements 🙂